src/materials/MeshPhongMaterial.js

```diff
@@ -94,6 +97,9 @@ function MeshPhongMaterial( parameters ) {
 	this.reflectivity = 1;
 	this.refractionRatio = 0.98;
 
+	this.gradientMap = null;
+	this.gradientMapAxisY = false;
```

Comment:
I'm also thinking that, maybe `MeshToonMaterial` could be more appropriate here... 😶